tests/test-cases/Group6-VIC-Machine/6-05-Create-Validation.robot

```diff
@@ -100,3 +100,41 @@ Trailing slash works as expected
     Log To Console  Installer completed successfully: %{VCH-NAME}...
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
+
+Whitelist registries
+    Set Test Environment Variables
+
+    Log To Console  \nInstalling VCH to test server...
+    # *.docker.io
+    ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target=%{TEST_URL}/ --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} ${vicmachinetls} --whitelist-registry *.docker.io
+    Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}  ${true}
+    # try a docker pull from docker.io; this should fail
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    Should Not Be Equal As Integers  ${rc}  0
+    Cleanup VIC Appliance On Test Server
```

Comment:
One of these failing will cause the whole list to not execute/fail.  Would prefer to break these up and move the cleanup into a teardown step so that we don't leak any VCHs.